HOME: /t3home/msaxer
USER: msaxer
SLURM_JOB_ID: 810724
HOSTNAME: t3wn35
CMSSW RELEASE BASE  /cvmfs/cms.cern.ch/slc7_amd64_gcc12/cms/cmssw/CMSSW_14_1_0_pre7
%MSG-i AlpakaService:  (NoModuleName) 04-Nov-2024 23:01:27 CET pre-events
AlpakaServiceSerialSync succesfully initialised.
Found 1 device:
  - Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz
%MSG
04-Nov-2024 23:01:56 CET  Initiating request to open file root://t3dcachedb.psi.ch:1094///pnfs/psi.ch/cms/trivcat/store/user/werdmann/store/relval/CMSSW_14_1_0_pre7/RelValTTbar_14TeV/GEN-SIM-RECO/141X_mcRun4_realistic_v1_STD_2026D110_PU-v1/2580000/d7a7c619-06d6-451a-a1fc-9ca3e17e4e27.root
[2024-11-04 23:01:56.574622 +0100][Error  ][XRootDTransport   ][136778] [t3dcachedb.psi.ch:1094.0] No protocols left to try
[2024-11-04 23:01:56.574831 +0100][Error  ][AsyncSock         ][136778] [t3dcachedb.psi.ch:1094.0] Socket error while handshaking: [FATAL] Auth failed
[2024-11-04 23:01:56.574934 +0100][Error  ][PostMaster        ][136778] [t3dcachedb.psi.ch:1094] elapsed = 0, pConnectionWindow = 31 seconds.
[2024-11-04 23:01:56.574963 +0100][Error  ][PostMaster        ][136778] [t3dcachedb.psi.ch:1094] Unable to recover: [FATAL] Auth failed.
[2024-11-04 23:01:56.574984 +0100][Error  ][XRootD            ][136778] [t3dcachedb.psi.ch:1094] Impossible to send message kXR_prepare (flags: kXR_stage , priority: 0, paths: //pnfs/psi.ch/cms/trivcat/store/user/werdmann/store/relval/CMSSW_14_1_0_pre7/RelValTTbar_14TeV/GEN-SIM-RECO/141X_mcRun4_realistic_v1_STD_2026D110_PU-v1/2580000/d7a7c619-06d6-451a-a1fc-9ca3e17e4e27.root). Trying to recover.
[2024-11-04 23:01:56.575085 +0100][Error  ][XRootD            ][136778] [t3dcachedb.psi.ch:1094] Impossible to send message kXR_query (code: kXR_Qconfig, arg length: 9). Trying to recover.
[2024-11-04 23:01:57.614280 +0100][Error  ][XRootDTransport   ][136778] [t3dcachedb.psi.ch:1094.0] No protocols left to try
[2024-11-04 23:01:57.614318 +0100][Error  ][AsyncSock         ][136778] [t3dcachedb.psi.ch:1094.0] Socket error while handshaking: [FATAL] Auth failed
[2024-11-04 23:01:57.614362 +0100][Error  ][PostMaster        ][136778] [t3dcachedb.psi.ch:1094] elapsed = 0, pConnectionWindow = 31 seconds.
[2024-11-04 23:01:57.614378 +0100][Error  ][PostMaster        ][136778] [t3dcachedb.psi.ch:1094] Unable to recover: [FATAL] Auth failed.
[2024-11-04 23:01:57.614390 +0100][Error  ][XRootD            ][136778] [t3dcachedb.psi.ch:1094] Impossible to send message kXR_open (file: /pnfs/psi.ch/cms/trivcat/store/user/werdmann/store/relval/CMSSW_14_1_0_pre7/RelValTTbar_14TeV/GEN-SIM-RECO/141X_mcRun4_realistic_v1_STD_2026D110_PU-v1/2580000/d7a7c619-06d6-451a-a1fc-9ca3e17e4e27.root, mode: 0660, flags: kXR_open_read kXR_async kXR_retstat ). Trying to recover.
%MSG-w XrdAdaptorInternal:  file_open 04-Nov-2024 23:01:57 CET pre-events
Failed to open file at URL root://t3dcachedb.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/werdmann/store/relval/CMSSW_14_1_0_pre7/RelValTTbar_14TeV/GEN-SIM-RECO/141X_mcRun4_realistic_v1_STD_2026D110_PU-v1/2580000/d7a7c619-06d6-451a-a1fc-9ca3e17e4e27.root?xrdcl.requuid=1f7d683d-7aeb-4a2d-89d2-eb28a6e1c2f4.
%MSG
[2024-11-04 23:01:57.647446 +0100][Error  ][XRootDTransport   ][136778] [t3dcachedb.psi.ch:1094.0] No protocols left to try
[2024-11-04 23:01:57.647478 +0100][Error  ][AsyncSock         ][136778] [t3dcachedb.psi.ch:1094.0] Socket error while handshaking: [FATAL] Auth failed
[2024-11-04 23:01:57.647516 +0100][Error  ][PostMaster        ][136778] [t3dcachedb.psi.ch:1094] elapsed = 0, pConnectionWindow = 31 seconds.
[2024-11-04 23:01:57.647526 +0100][Error  ][PostMaster        ][136778] [t3dcachedb.psi.ch:1094] Unable to recover: [FATAL] Auth failed.
[2024-11-04 23:01:57.647534 +0100][Error  ][XRootD            ][136778] [t3dcachedb.psi.ch:1094] Impossible to send message kXR_open (file: /pnfs/psi.ch/cms/trivcat/store/user/werdmann/store/relval/CMSSW_14_1_0_pre7/RelValTTbar_14TeV/GEN-SIM-RECO/141X_mcRun4_realistic_v1_STD_2026D110_PU-v1/2580000/d7a7c619-06d6-451a-a1fc-9ca3e17e4e27.root?tried=, mode: 0660, flags: kXR_open_read kXR_async kXR_retstat ). Trying to recover.
%MSG-w XrdAdaptorInternal:  file_open 04-Nov-2024 23:01:57 CET pre-events
Failed to open file at URL root://t3dcachedb.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/werdmann/store/relval/CMSSW_14_1_0_pre7/RelValTTbar_14TeV/GEN-SIM-RECO/141X_mcRun4_realistic_v1_STD_2026D110_PU-v1/2580000/d7a7c619-06d6-451a-a1fc-9ca3e17e4e27.root?tried=&xrdcl.requuid=4d77f9c6-277f-44e0-b2db-98a125c45719.
%MSG
----- Begin Fatal Exception 04-Nov-2024 23:01:57 CET-----------------------
An exception of category 'FileOpenError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing input source of type PoolSource
   [2] Calling RootInputFileSequence::initTheFile()
   [3] Calling StorageFactory::open()
   [4] Calling XrdFile::open()
Exception Message:
Failed to open the file 'root://t3dcachedb.psi.ch//pnfs/psi.ch/cms/trivcat/store/user/werdmann/store/relval/CMSSW_14_1_0_pre7/RelValTTbar_14TeV/GEN-SIM-RECO/141X_mcRun4_realistic_v1_STD_2026D110_PU-v1/2580000/d7a7c619-06d6-451a-a1fc-9ca3e17e4e27.root'
   Additional Info:
      [a] Input file root://t3dcachedb.psi.ch:1094///pnfs/psi.ch/cms/trivcat/store/user/werdmann/store/relval/CMSSW_14_1_0_pre7/RelValTTbar_14TeV/GEN-SIM-RECO/141X_mcRun4_realistic_v1_STD_2026D110_PU-v1/2580000/d7a7c619-06d6-451a-a1fc-9ca3e17e4e27.root could not be opened.
      [b] XrdCl::File::Open(name='root://t3dcachedb.psi.ch//pnfs/psi.ch/cms/trivcat/store/user/werdmann/store/relval/CMSSW_14_1_0_pre7/RelValTTbar_14TeV/GEN-SIM-RECO/141X_mcRun4_realistic_v1_STD_2026D110_PU-v1/2580000/d7a7c619-06d6-451a-a1fc-9ca3e17e4e27.root', flags=0x10, permissions=0660) => error '[FATAL] Auth failed: No protocols left to try' (errno=0, code=204). No additional data servers were found.
      [c] Last URL tried: root://t3dcachedb.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/werdmann/store/relval/CMSSW_14_1_0_pre7/RelValTTbar_14TeV/GEN-SIM-RECO/141X_mcRun4_realistic_v1_STD_2026D110_PU-v1/2580000/d7a7c619-06d6-451a-a1fc-9ca3e17e4e27.root?tried=&xrdcl.requuid=4d77f9c6-277f-44e0-b2db-98a125c45719
      [d] Problematic data server: t3dcachedb.psi.ch:1094
      [e] Disabled source: t3dcachedb.psi.ch:1094
----- End Fatal Exception -------------------------------------------------
cp: cannot stat 'pv.root': No such file or directory
